README.md

```diff
@@ -58,7 +57,34 @@ deployment options.
 }
 ```
 
-### Adding new integrations
+#### Running storage operations locally
+If you need to run storage (S3, GCS or Azure Blob) operations locally you can run the agent setting these 
+environment variables: `MCD_STORAGE` and `MCD_STORAGE_BUCKET_NAME`, for example:
+```shell
+PYTHONPATH=. MCD_DEBUG=true MCD_STORAGE_BUCKET_NAME=agent-bucket MCD_STORAGE=GCS python apollo/interfaces/generic/main.py
+```
+Please note this needs your environment to be configured with credentials for the environment hosting the bucket, 
+for GCS you need to login using `gcloud` and for AWS you need to specify the required environment variables for 
+`boto3` to connect to the bucket.
+
+Storage operations are executed using the special connection type: `storage`, for example to list all objects
+in the bucket you can send a POST to http://localhost:8081/api/v1/agent/execute/storage/list-objects
+with the following body:
+```json
+{
+    "operation": {
+        "trace_id": "1234",
+        "commands": [
+            {
+                "method": "list_objects"
+            }
+        ]
+    }
+}
+```
+
+## Adding new integrations
+How to add new integrations is documented in Notion [here](https://www.notion.so/montecarlodata/Adding-support-for-Remote-Agents-to-integrations-3d5025ef36eb47de8488cdafdc39d42c?pvs=4). 
```

Comment:
Given this is a public repo, we probably shouldn't be adding notion links here. Maybe that doc could be moved to the repo.